org-formation/organization.yaml

```diff
@@ -500,3 +507,17 @@ Organization:
       Alias: org-sagebase-saturncloud-dev
       Tags:
         <<: !Include ./_default_org_tags.yaml
+
+  #------------ Personal Accounts -----------
+  VYadavAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-vyadav
+      RootEmail: vijay.yadav@sagebase.org
```

Comment:
@vjbytes102 do you have any AWS accounts registered to this email?

Reply:
No